content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -333,7 +333,13 @@ using `allowedTopologies`.
 
 {{< note >}}
 `zone` and `zones` parameters are deprecated and replaced with
-[allowedTopologies](#allowed-topologies)
+[allowedTopologies](#allowed-topologies). When
+[GCE CSI Migration](/docs/concepts/storage/volumes/#gce-csi-migration) is
+enabled, a GCE PD volume can be provisioned in a topology that does not match
+any nodes, but any pod trying to use that volume will fail to schedule. With
+legacy pre-migration GCE PD, in this case an error will be produced
```

Comment:
Clarify: What is legacy pre-migration GCE PD?

When GCE CSI Migration is enabled, you can provision a GCE PD volume in a topology that does not match any nodes. Any pod trying to use the GCE PD volume fails to schedule. If you provision a legacy, pre-migration GCE PD volume, an error occurs.